recipes/openpmd-viewer/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "openpmd-viewer" %}
+{% set version = "0.9.1" %}
+{% set build = 0 %}
+{% set sha256 = "ac3b2fd00b82fc4ecb2ac7ae991d09fdef3068b0c5e54a08d832c2b7ed65fe6b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/openPMD/openPMD-viewer/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - cython
+    - numpy
+    - setuptools
+  host:
+    - python
+    - pip
+  run:
+    - h5py
+    - matplotlib
```

Comment:
Please use `matplotlib-base` to avoid hard dependencies on Qt.

Reply:
Great idea, thanks!